src/CommentionsServiceProvider.php

```diff
@@ -41,14 +41,14 @@ public function packageBooted(): void
             [
                 Js::make('commentions-scripts', __DIR__ . '/../resources/dist/commentions.js'),
             ],
-            'commentions'
+            'kirschbaum-development/' . static::$name
```

Comment:
I added this, not sure how you feel about it. It's recommended in the FIlament documentation for packages to do this.

Reply:
Yep, sounds like a change we should do.